Makefile

```diff
@@ -21,6 +21,7 @@ JEKYLL_ENV ?= development
 SITE_BUILD_CMD := $(CONTAINER_RUN_CMD) --rm -i -u "`id -u`:`id -g`" \
 	$(shell [ -t 0 ] && echo '-t') \
 	-e JEKYLL_ENV=$(JEKYLL_ENV) \
+	-e JEKYLL_GITHUB_TOKEN="$$JEKYLL_GITHUB_TOKEN" \
```

Comment:
```suggestion
	-e JEKYLL_GITHUB_TOKEN="$JEKYLL_GITHUB_TOKEN" \
```

Reply:
This is passed to the make target (to be executed). It takes it from the env (not from a makefile variable) so we need the double  symbol

Reply:
Do'h, makes sense.